src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -384,6 +384,12 @@ export class DBTCloudProjectIntegration
     return this.python.connected;
   }
 
+  async cleanupConnections(): Promise<void> {
+    try {
+      await this.python.ex`project.cleanup_connections()`;
+    } catch (_) {}
+  }
```

Comment:
This won't work on dbt cloud as there is no access to the python env in this case.